features/bookmarks.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Cite and Email Bookmarks
+  As a logged in user,
+  I would like to bookmark certain records
+  and be able to share them 
+  by citing and emailing them.
+
+@bm
```

Comment:
@eshadatta No reason to tag unless you're using Around, Before, etc or want to group or skip. 

If you do need to tag this please use a descriptive tag like `@bookmark` instead of the cryptic `@bm`